src/iframe-helper.js

```diff
@@ -36,6 +36,9 @@ export function listen(iframe, typeOfMessage, callback, opt_is3P) {
   const sentinel = getSentinel_(opt_is3P);
   let unlisten;
   let listener = function(event) {
+    const data = typeof event.data == 'string'
+                ? JSON.parse(event.data)
```

Comment:
Please file an issue to parse only once per received message (not once per listener).